src/libs/canvas.js

```diff
@@ -138,24 +138,24 @@ function setURL(db, user, url, callback) {
 }
 
 /**
- * Retrieves a user's events on Canvas
- * @function getEvents
+ * Retrieves a user's events on Canvas from their URL
+ * @function getUserCal
  *
  * @param {Object} db - Database connection
  * @param {string} user - Username to get schedule
- * @param {getEventsCallback} callback - Callback
+ * @param {getUserCalCallback} callback - Callback
  */
 
 /**
  * Returns a user's schedule for that day
- * @callback getEventsCallback
+ * @callback getUserCalCallback
  *
  * @param {Object} err - Null if success, error object if failure.
  * @param {Boolean} hasURL - Whether user has set a valid portal URL. Null if failure.
  * @param {Object} events - Array of all the events in the month. Null if failure.
  */
 
-function getEvents(db, user, callback) {
+function getUserCal(db, user, callback) {
```

Comment:
On line 227 of `canvas.js`, there's a property for every canvas event called `canvas: true`. We may be able to get rid of that because that's also being stored in the cache, which is a bit redundant. We might even be able to get rid of that property completely since the front-end now fetches Canvas and MyMICDS events separately. However, for now, it might be best to just add the `canvas: true` property which returning the cache.